team.json

```diff
@@ -111,34 +92,28 @@
           "country": "CY"
         },
         {
-          "firstName": "Michael “Furyo” Cabilao",
+          "firstName": "Daniel “IllustrisJack” Handayan Mayer",
```

Comment:
?

Reply:
You were moved from Project Management to Development as far as I can see, is this info wrong? :O

Reply:
yea i´m not an active member of the development team per say

Reply:
maybe external consultant at best haha :D